cli/command/user/create.go

```diff
@@ -99,7 +99,7 @@ func runCreate(storageosCli *command.StorageOSCli, opt createOptions) error {
 	params := types.UserCreateOptions{
 		Username: opt.username,
 		Password: opt.password,
-		Groups:   opt.groups,
+		Groups:   []string(opt.groups),
```

Comment:
I think you're dragging in a PR we don't want with this - it is part of #159 (https://github.com/storageos/go-cli/pull/159/files#diff-92439c9d6c61df31e250cf6b5f992ce4).

Reply:
I double checked it and it is because the array groups PR was merged to both go-api and go-cli, but then reverted only in go-cli, so the code cannot build without this conversion if go-cli updates its go-api to master or any new feature branch.